lib/ansible/module_utils/common/collections.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c), Sviatoslav Sydorenko <ssydoren@redhat.com> 2018
+# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
+"""Collection of low-level utility functions."""
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+# Note: When getting Sequence from collections, it matches with strings.  If
+# this matters, make sure to check for strings before checking for sequencetype
+try:
+    """Python 3.3+ branch."""
+    from collections.abc import KeysView, Sequence
+except ImportError:
+    """Use old lib location under 2.6-3.2."""
+    from collections import KeysView, Sequence
+
+SEQUENCETYPE = frozenset, KeysView, Sequence
```

Comment:
Don't do this.  It's "impure".  We should make is_sequence() apply purely to Sequences rather than mixing in KeysView and frozensets as well.  (We probably want both is_sequence and is_iterator where some use cases should be ported to is_iterator from whatever they are now).

Reply:
@abadger I've reverted logical changes for now

Reply:
do you want me to add is_iterator in this PR or separate one?

Reply:
You are right... is_iterable which checks that an object is iterable is probably more useful than something that checks if an object is an iterator.